data_source_examples/aha/using_pyairtable/example.py

```diff
@@ -0,0 +1,127 @@
+import os
+import requests
+from pyairtable import Table
+from dotenv import load_dotenv
+
+load_dotenv()
+
+# Configuration variables for Airtable and Snowflake
+AHA_ACCOUNT_NAME = os.environ['AHA_ACCOUNT_NAME']
+AHA_API_KEY = os.environ['AHA_API_KEY']
+AIRTABLE_API_KEY = os.environ['AIRTABLE_API_KEY']
+AIRTABLE_BASE_ID = os.environ['AIRTABLE_BASE_ID']
+AIRTABLE_TABLE_ID = os.environ['AIRTABLE_TABLE_ID']
+AIRTABLE_UNIQUE_FIELD_NAME = os.environ['AIRTABLE_UNIQUE_FIELD_NAME']
+
+#Initialize Airtable Client
+Table = Table(AIRTABLE_API_KEY, AIRTABLE_BASE_ID, AIRTABLE_TABLE_ID)
+
+# Retrieve all features from Aha! account
+endpoint = 'https://'+AHA_ACCOUNT_NAME+'.aha.io/api/v1/features'
```

Comment:
last thing, but might be easier to use f-strings instead of concatenating: 

endpoint = f'https://{AHA_ACCOUNT_NAME}.aha.io/api/v1/features'
headers = {"Authorization": f"Bearer {AHA_API_KEY}"}